src/test/java/org/prebid/server/auction/BidResponseCreatorTest.java

```diff
@@ -747,10 +760,40 @@ public void shouldAddExtPrebidEvents() {
         verify(cacheService, never()).cacheBidsOpenrtb(anyList(), anyList(), any(), any(), any());
     }
 
+    @Test
+    public void shouldNotAddExtPrebidEventsIfExtRequestPrebidEventsNull() {
```

Comment:
Pls rename to `shouldNotAddExtPrebidEventsIfExtRequestPrebidEventsIsNull`.
Also, add the additional unit test when Account has `eventsEnabled = false`.